pkgs/top-level/python-packages.nix

```diff
@@ -1467,6 +1467,11 @@ in {
 
   ckcc-protocol = callPackage ../development/python-modules/ckcc-protocol { };
 
+  inherit (import ../development/python-modules/clang self)
```

Comment:
As we want to get rid of multiple versions as soon as possible I don't think we should convert this into a common package definition. Just copy the current file and adjust it for the older version or only use overrideAttr if possible.

Also I am not sure if we even can/want to introduce multiple version of the same package.

/cc @dotlambda

Reply:
For better or worse, tensorflow 2.6.0 depends on clang v5. The currently packaged version is v11, so I didn't want to break compatibility for people already using v11.

Reply:
Happy to package it any way that you feel is most appropriate as long as tensorflow can get access to clang_5.